tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -86,6 +86,56 @@ const RETURN_PAREN_NEW_LINE = `
   });
 `;
 
+const RETURN_PAREN_NEW_LINE_OPENING = `
+  var Hello = createReactClass({
+    render: function() {
+      return (
+
```

Comment:
These extra newlines here seem odd - why is this allowed to be here?

Reply:
See my earlier comment: 
https://github.com/yannickcr/eslint-plugin-react/pull/1984#issue-215844164

My first attempts at fixing this caused extraneous newlines to disappear which seemed like an undesired side-effect of this particular rule. The extra newline isn't being added by the fix -- its in the original test condition. The test is making sure the extra newline is _not_ being stripped.

Reply:
That makes sense (although I’d want it stripped, this probably isn’t the rule for it)